server/lib/src/credential/mod.rs

```diff
@@ -505,6 +505,15 @@ impl Credential {
         })
     }
 
+    //TODO: Add validation for securitykey label
```

Comment:
These will be removed soon, so don't worry about it.

Reply:
Ah, alright I'll remove that method then.